src/api/app/controllers/webui/staging_workflows/request_ignorations_controller.rb

```diff
@@ -0,0 +1,60 @@
+module Webui
+  module StagingWorkflows
+    class RequestIgnorationsController < WebuiController
+      layout 'webui2/webui'
+
+      before_action :require_login
+      before_action :set_bootstrap_views
+      before_action :set_staging_workflow
+      before_action :set_request_ignoration, only: [:destroy]
+      after_action :verify_authorized, except: [:index]
+
+      def index
+        @request_ignorations = @staging_workflow.request_ignorations
+      end
+
+      def create
+        authorize @staging_workflow
+
+        request_ignoration = @staging_workflow.request_ignorations.build(request_ignoration_params)
+
+        if request_ignoration.save
+          flash[:success] = "Request #{request_ignoration.bs_request_id} was successfully ignored"
```

Comment:
[nitpick] It may be worthwhile to define a `to_s` method in `BsRequest`:

```ruby
def to_s
  id.to_s
end
```

so that you can save the `_id` every time you interpolate :nerd_face:

Reply:
it should be noted that bsrequests are known by their number not by their id to users

Reply:
Being a nitpick, and taking into account @coolo and @bgeuken 's comments I will leave like it is, without adding the `.to_s` method in `BsRequest`.

Reply:
I meant:

```ruby
def to_s
  number.to_s
end
```